allofplos/gdrive.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import os
+import tarfile
+from zipfile import ZipFile, BadZipFile
+
+import requests
+from tqdm import tqdm
+
+from . import get_corpus_dir
+
+# Variables needed
+zip_id = '0B_JDnoghFeEKLTlJT09IckMwOFk'
+metadata_id = '0B_JDnoghFeEKQUhKWXBOVy1aTlU'
+local_zip = 'allofplos_xml.zip'
+zip_metadata = 'zip_info.txt'
+time_formatting = "%Y_%b_%d_%Hh%Mm%Ss"
+min_files_for_valid_corpus = 200000
```

Comment:
is this arbitrary?

Reply:
A cut-off to avoid too much scraping journal pages for new articles. You could conceivably download one article and then run `plos_corpus` and download the other 227000 manually. This assures that they have enough articles from the zip that it won't be too costly otherwise, though technically the number is abitrary